Autopilot/PathManager/Src/landingManager.cpp

```diff
@@ -0,0 +1,200 @@
+#include "waypointManager.hpp"
+#include "landingManager.hpp"
+#include <cmath>
+
+const int SET_THROTTLE_OFF = -1000;
+const double DISTANCE_OF_LANDING = 10; //in meters
+const int METERS_PER_DEG_LAT = 111320;
+const int ANGLE_OF_LANDING = 5; //in degrees
+
+double LandingManager::changingAltitude(_WaypointManager_Data_In * input, _PathData * aimingPoint, _PathData * intersectionPoint, _PathData * stoppingPoint)
+{
+    //arrays to hold the important points
+    float aPoint[3]; //aiming point
+    float iPoint[3]; //intersection point
+    float sPoint[3]; //stopping point
+    float cPoint[3]; //current point
+    float projectedPoint[3];
```

Comment:
Is there a reason why float arrays are used here instead of a struct?

Reply:
Is there any benefit in using a struct? I think in other files vectors were represented in arrays, that's why I used it here.